src/docs/tool-libfuzzer.dox

```diff
@@ -0,0 +1,210 @@
+/*! @page tool-libfuzzer Testing with LLVM LibFuzzer
+
+# Building and running an LLVM LibFuzzer target
+
+LLVM LibFuzzer is in-process, coverage-guided, evolutionary fuzzing engine. It feeds a series of
+fuzzed inputs via a "target" function and attempts to trigger crashes, memory bugs and undefined
+behavior.
+
+## Step 1: Select Clang as your C compiler in your build configuration
+
+Support for LibFuzzer is implemented as a compiler flag in Clang. Our configuration checks whether
+the compiler in use supports \c -fsanitize=fuzzer and if so, elects to build the fuzz targets.
+
+Compiling with the ASan isn't mandatory but it is recommended since fuzzing often exposes memory
+bugs.
+
+@code
+$ cd build_posix/
+$ ../configure CC="clang-8" CFLAGS="-fsanitize=address"
+@endcode
+
+## Step 2: Build as usual
+
+@code
+$ cd build_posix
+$ make
+@endcode
+
+## Step 3: Run a fuzz target
+
+Each fuzz target is just a program under \c build_posix/test/fuzz/. Once you become familiar with
```

Comment:
7a17f05

Addressed wording comments.

Happy to be convinced if there's an upside that I'm not considering but I don't see a list of targets in the doc as helpful. Realistically, it's not going to be kept up to date with what's in the repo because people are going to forget. And if we're only going to describe the first two and not keep it up to date, then it's even less useful.